org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/manager/model/IMethodReturnValue.java

```diff
@@ -33,5 +33,5 @@
  */
 package org.openjdk.jmc.console.ext.agent.manager.model;
 
-public interface IMethodReturnValue extends ICapturedValue {
+public interface IMethodReturnValue extends INamedCapturedValue {
```

Comment:
I looked at the returnValue agent object and it seems you can set a name.  Also the wizard page would then get more complicated because I would have to return the "Name" input field if the user chooses a return value.  In this case it might be better to make two separate parameter and return value wizards because they have a few differences now.

Reply:
A second look at the agent implementation: yes, I agree with you. The `name` property should be configurable.

While the schema definition file doesn't allow a `<name>` property and will throw an error upon validation, it's a bug not catch during the [reviewing process](https://github.com/openjdk/jmc/pull/66) of [JMC-6725](https://bugs.openjdk.java.net/browse/JMC-6725).

There should be an upstream issue created to address this problem. We can just leave this at it is for now, until it gets resolved upstream.

Reply:
#61 is created to track this problem